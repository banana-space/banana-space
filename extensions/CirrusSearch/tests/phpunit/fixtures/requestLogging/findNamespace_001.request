{
	"_comment": "known namespace",
	"type": "findNamespace",
	"name": "file:"
}
