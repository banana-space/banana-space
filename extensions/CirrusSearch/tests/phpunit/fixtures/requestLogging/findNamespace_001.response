[
    {
        "took": 2,
        "timed_out": false,
        "_shards": {
            "total": 4,
            "successful": 4,
            "failed": 0
        },
        "hits": {
            "total": 1,
            "max_score": 1.6931472,
            "hits": [
                {
                    "_index": "wiki_general_first",
                    "_type": "namespace",
                    "_id": "6",
                    "_score": 1.6931472
                }
            ]
        }
    }
]
