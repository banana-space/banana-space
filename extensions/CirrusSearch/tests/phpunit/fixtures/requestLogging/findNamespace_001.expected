[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits_offset": 0,
                    "hits_returned": 1,
                    "hits_total": 1,
                    "indices": [
                        "mw_cirrus_metastore"
                    ],
                    "max_score": 1.6931472,
                    "query": "file",
                    "query_type": "namespace",
                    "search_time_ms": 2,
                    "suggestion_requested": false
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 1,
            "hitsTotal": 1,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "mw_cirrus_metastore",
            "maxScore": 1.6931472,
            "namespaceName": "file",
            "query": "file",
            "queryType": "namespace",
            "source": "cli",
            "suggestionRequested": false,
            "tookMs": 0
        },
        "level": "debug",
        "message": "lookup namespace for {namespaceName} against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]